### ISSUE TITLE
Review Requirements and Deadlines

### ISSUE THREAD

**Issue by nathanRomad** [nathanRomad/topFIVE#5]
**Technical Requirements:**
- React Application
- Single Page Application,
- CRUD using JS Fetch API
-Login: Users must be able to authenticate
- Users and User dependent data
- Persistent data storage (i.e., json-server)
Your database must look like the ERD your mentor approved!!!
- All primary keys and foreign keys in your database need to be integers
- The console in your developer tools should not have any errors, warnings are ok.
- Your application should be data driven. This means no hard-coded values for dynamic routes, drop-downs, etc. These should always be fetched from the database.
- It is NOT required to use an external API.
- Minimal Styling

**Planning:**
- Use dbdiagram.io, Lucidchart, etc. for ERD.
- Create wireframes (using moqups, lucidchart, photoshop, adobe XD, Figma, etc.)
- Issue tickets on Github must be written following BDD.
- If you are using an external API, gather all sample data and be prepared to demo your api at your one on one (via Postman) before committing to use the API.
- If you want to use to use something we haven’t covered in class, you have to prove that you’ve implemented the code before your proposal will be accepted. If it is accepted, you will be given lower priority if you need help on that technology.
- Make sure you have a project board set up in github and migrate your issue tickets.
- Once your ERD is approved, NO CHANGES SHOULD BE MADE TO IT without your capstone mentors approval.
- You should not start writing any code until you have had your one on one with your capstone mentor and they have approved your full proposal. If you have not fleshed out your planning enough, your capstone mentor will not let you code.

**Project NoNos:**
- No MVP can include a calendar. Do not try to integrate with Google Calendar (or anything else).
- No Web Sockets. Yeah, we know it’s cool. It’s also a tremendous obstacle and provides absolutely zero value to solving your problem.
- No games without approval.
